DevEdu.API/DevEdu.DAL/Repositories/BaseRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Data;
+using System.Data.SqlClient;
+
+namespace DevEdu.DAL.Repositories
+{
+    public abstract class BaseRepository
+    {
+        protected const string ConnectionString =
+            @"Data Source=80.78.240.16;Initial Catalog = DevEdu; Persist Security Info=True;User ID = student;Password=qwe!23;";
+
+        protected IDbConnection _connection;
+        protected string _insertProcedure;
```

Comment:
please remove these fields and create consts in Repositories (as we did it in a lesson)